docs/documentation/getting-started.md

```diff
@@ -98,193 +34,52 @@ accompany your sessions.
-
-## Disclaimer
-
-Saros/I does not include sub-modules when sharing a module (see [Module Restrictions](#module-restrictions)).
-As a consequence, such sub-modules might not be present for all session participants.
-If a participant deletes a shared directory that contains a sub-module in the local setup of another participant, this sub-module will be deleted without any notice.
-
-Furthermore, there are still some known bugs in the current release. Please have a look at the section [Known Bugs](#known-bugs).
-
-## Features
-
-This alpha version provides most of the basic functionality of Saros.
-You can
-
-- add existing XMPP-accounts
-- start a session with another person
-  - Sessions in Saros/I are currently limited to two participants (host and one client)
-- share exactly one module through Saros; the shared module must meet the restrictions described [here](#module-restrictions)
-- transfer the initial content of the module shared by the host to all participating clients
-- work on shared resources
-- create, delete, and move resources in the shared module
-- interact freely with non-shared resources
-- follow other participants of the session ([follow mode](features.md#follow-mode))
-
+Most interactions with the Saros session logic (like starting or ending a session) can be done through the Saros tool window.
+This window is attached to the bottom right of the IDE by default.
+It is marked with the title "Saros" and the Saros icon (![saros icon](images/icons-i/saros.png)).
+All actions described in the following sections take place in this tool window.
 
-### Sharing Complex Modules
+{% endcapture %}
+{% include ide-tabs.html eclipse=eclipse intellij=intellij %}
 
-Even though Saros offers the option to create the module on the client side as part of the session, this should only be used for relatively simple modules.
-For more complex modules, it is advised to share the module structure some other way (e.g. a VCS) before starting a session. Saros currently does not set up things like libraries or module dependencies, meaning they would have to be configured by hand.
+## Start a Session
 
-## Compatibility
+In the following we show you how to set up Saros to start your first session.
 
-The current release `0.2.2` is not compatible with the previous Saros/I releases (`0.1.0`, `0.2.0`, or `0.2.1`) or other Saros plugins (like Saros/E).
+### Step 1: Creating an XMPP Account
 
+You first need an **XMPP account** to use Saros.
 
-## How to Use Saros/I
+1.  You can **create a new account**:
+  * On our XMPP server using our **[signup page](https://saros-con.imp.fu-berlin.de:5280/register/new)**
+        * Note: Due to abuse by spammers, we had to disable in-band registration and from within Saros.
+  * On other XMPP servers as [jabber.org](https://www.jabber.org/) ([register.jabber.org](https://register.jabber.org/)).
+2.  You can **use an existing account**
+3.  You [setup your own XMPP server](setup-xmpp.md).
```

Comment:
This should be a list instead of an enumeration.
```suggestion
*  You can **create a new account**:
  * On our XMPP server using our **[signup page](https://saros-con.imp.fu-berlin.de:5280/register/new)**
        * Note: Due to abuse by spammers, we had to disable in-band registration and from within Saros.
  * On other XMPP servers as [jabber.org](https://www.jabber.org/) ([register.jabber.org](https://register.jabber.org/)).
*  You can **use an existing account**
*  You [setup your own XMPP server](setup-xmpp.md).
```